.github/workflows/update-parity-tracker.yml

```
@@ -1,19 +1,39 @@
 name: Update Feature Parity Tracker
 
 on:
+  pull_request:
+    paths: &watched
```

actionlint (1.7.12)

[error] anchor "watched" is defined but not used

(syntax-check)